pv_site_api/main.py

```diff
@@ -291,6 +292,51 @@ def get_pv_forecast_many_sites(
     return forecasts
 
 
+@app.get("/sites/{site_uuid}/clearsky_estimate")
+def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_session)):
+    """
+    ### Gets a estimate of AC production under a clear sky
+    """
+    if int(os.environ["FAKE"]):
+        fake_sites = make_fake_site()
+        site = fake_sites.site_list[0]
+    else:
+        site = get_site_by_uuid(session, site_uuid)
+    loc = location.Location(site.latitude, site.longitude)
+
+    # Create DatetimeIndex over four days, with a frequency of 15 minutes.
+    # Starts from midnight yesterday.
+    times = pd.date_range(start=get_yesterday_midnight(), periods=384, freq="15min", tz="UTC")
+    clearsky = loc.get_clearsky(times)
+    solar_position = loc.get_solarposition(times=times)
+
+    # Using default tilt of 0 and orientation of 180 from defaults of PVSystem
+    tilt = site.tilt if site.tilt is not None else 0
+    orientation = site.orientation if site.orientation is not None else 180
+
+    irr = irradiance.get_total_irradiance(
+        surface_tilt=tilt,
+        surface_azimuth=orientation,
+        dni=clearsky["dni"],
+        ghi=clearsky["ghi"],
+        dhi=clearsky["dhi"],
+        solar_zenith=solar_position["apparent_zenith"],
+        solar_azimuth=solar_position["azimuth"],
+    )
+    # @TODO: allow differing inverter and module capacities
```

Comment:
Would you mind opening an issue for this? and add the link to this comment. I always worry that we end up with lots of TODOs with no actions from them

Reply:
yep—done!